iotdb-core/datanode/src/main/java/org/apache/iotdb/db/pipe/receiver/airgap/IoTDBAirGapReceiver.java

```diff
@@ -117,12 +118,18 @@ private void receive() throws IOException {
       final ByteBuffer byteBuffer = ByteBuffer.wrap(data, LONG_LEN, data.length - LONG_LEN);
 
       // Pseudo request, to reuse logic in IoTDBThriftReceiverAgent
+      byte version = ReadWriteIOUtils.readByte(byteBuffer);
+      short type = ReadWriteIOUtils.readShort(byteBuffer);
+      if (type == PipeRequestType.HANDSHAKE_V2.getType()) {
```

Comment:
What about only serialize the useful contents instead of get the buffer length here?